infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftInfraModule.java

```diff
@@ -107,6 +101,7 @@ protected void configure() {
 
     Multibinder<NamespaceConfigurator> namespaceConfigurators =
         Multibinder.newSetBinder(binder(), NamespaceConfigurator.class);
+    namespaceConfigurators.addBinding().to(UserPermissionConfigurator.class);
```

Comment:
wondering why this configurator was not applied before? from what I see it falls back on `che.infra.kubernetes.user_cluster_roles` property. In the context of https://issues.redhat.com/browse/CRW-3328 it is not clear for me how this would help

Reply:
To be honest I don't know as well.

Reply:
so, do we really need this change? e.g. why would you like to bind this configurator as part of this PR?